### PR TITLE
Wireframe mode

### PR DESCRIPTION
**Pull request by atharva-18**
# 🎉 New feature

## Summary
View models and links as wireframes, similar to Gazebo Classic.
wireframe_ign_f

Nested model
wireframe_nested_model

## Test it
Test it by running `ign gazebo -v 4 examples/worlds/nested_model.sdf`

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [x] `codecheck` passed (See [contributing](https://ignitionrobotics.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+repo%3Aosrf%2Fsdformat+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge**

🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸

### CODE REVIEW

File: src/gui/plugins/modules/EntityContextMenu.qml [hunk 1]
```diff
@@ -63,6 +63,14 @@ Item {
     id: secondMenu
     x: menu.x + menu.width
     y: menu.y + viewSubmenu.y
+    MenuItem {
```
Comment (nkoenig) [inline src/gui/plugins/modules/EntityContextMenu.qml:66]: Can you move this menu item below `Collsions` so that they are alphabetical?

Comment (atharva-18) [reply src/gui/plugins/modules/EntityContextMenu.qml]: Changed in [`1393204`](https://github.com/ignitionrobotics/ign-gazebo/pull/816/commits/139320417f89a01a1b77f7b2d0117a8d0f6990da)

File: src/gui/plugins/scene3d/Scene3D.cc [hunk 2]
```diff
@@ -811,6 +817,33 @@ void IgnRenderer::Render()
...
+        ignerr << "Unable to find node name ["
+               << this->dataPtr->viewWireframesTarget
+               << "] to view wireframes" << std::endl;
+      }
+
+      this->dataPtr->viewWireframesTarget.clear();
+    }
+  }
+
+
```
Comment (ahcorde) [inline src/gui/plugins/scene3d/Scene3D.cc:846]:
```suggestion
```

**Review by ahcorde — APPROVED** [review]
LGTM, just a style comment

**Comment by ahcorde** [comment]
@atharva-18 it seems that you were able to make it work with nested models. Do you mind to have a look to this issue? https://github.com/ignitionrobotics/ign-gazebo/issues/767 . Collision visualization is not working right now with nested model.

**Review by nkoenig — APPROVED** [review]

**Comment by osrf-triage** [comment]
This pull request has been mentioned on **Gazebo Community**. There might be relevant details there:

https://community.gazebosim.org/t/google-summer-of-code-2021-new-gui-widgets-in-ignition-gazebo/1081/1

**Pull request merged**